CONTRIBUTING.md

```diff
@@ -74,10 +74,11 @@ Small pull requests are much easier to review and more likely to get merged.
 
 ### Installation
 
-1. Ensure you have [pnpm](https://pnpm.io/installation) installed
-1. After cloning the repository, run `pnpm install`. You can do this in the root directory or in the `svelte` project
-1. Move into the `svelte` directory with `cd packages/svelte`
-1. To compile in watch mode, run `pnpm dev`
+Ensure you have [pnpm](https://pnpm.io/installation) installed. After cloning the repository, run `pnpm install`.
+
+### Developing
+
+To build the UMD version of `svelte/compiler` (this is only necessary for CommonJS consumers, or in-browser use), run `pnpm build` inside `packages/svelte`. To rebuild whenever source files change, run `pnpm dev`.
```

Comment:
in-browser probably isn't exactly true since you can run ESM in the browser
```suggestion
To build the UMD version of `svelte/compiler` (this is only necessary for CommonJS consumers and the REPL), run `pnpm build` inside `packages/svelte`. To rebuild whenever source files change, run `pnpm dev`.
```

Reply:
I think 'in-browser' is valid. I don't think there's a reasonable way currently to run an unbundled copy of the compiler in the browser.